x-pack/plugins/security_solution/public/common/components/alerts_treemap_panel/index.test.tsx

```diff
@@ -232,6 +255,25 @@ describe('AlertsTreemapPanel', () => {
     await waitFor(() => expect(screen.getByTestId('progress')).toBeInTheDocument());
   });
 
+  it('does NOT render the progress bar when loading is true, but the panel is collapsed', async () => {
```

Comment:
A network request won't be made when the panel is in the collapsed state, but the component still gets `loading: true` when the page-level refresh button is clicked.